### PR TITLE
fix(writer): Add EBUSY to transient errors on Linux

### PR DESCRIPTION
**Pull request by jhermsmeier**
Change-Type: patch

### CODE REVIEW

**Comment by lurch** [comment]
Are we sure this is the right fix? The previous implementation in `etcher-image-write` didn't seem to ignore EBUSY errors on Linux https://github.com/resin-io-modules/etcher-image-write/blob/master/tests/filesystem.spec.js#L399 ?

File: lib/writer/error-types.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ module.exports = {
     if (process.platform === 'darwin') {
       return error.code === 'ENXIO' || error.code === 'EBUSY'
     } else if (process.platform === 'linux') {
-      return error.code === 'EIO'
+      return error.code === 'EIO' || error.code === 'EBUSY'
```
Comment (lurch) [inline lib/writer/error-types.js]: If this **is** what we want to do, then the comment on line 27 needs updating ;-)

**Comment by jviotti** [comment]
@jhermsmeier Did you study some of the Sentry instances of this error? Were you able to reproduce? Do we know for sure EBUSY means that we lost access to the drive?

File: lib/writer/error-types.js (hunk 1 above)
Comment (lurch) [reply lib/writer/error-types.js]: bump ;0

Comment (jhermsmeier) [reply lib/writer/error-types.js]: Fixed

**Pull request merged**